### ISSUE TITLE
[FEAT] Frozen Backbone Finetuning

### ISSUE THREAD

**Issue by Jaster1999** [lightly-ai/lightly-train#615]
Is it possible to train models (e.g. ltdetr) with dinov3 backbones but keep the features frozen for the backbone (and turn off gradients). In the Dinov3 paper they report good accuracy with frozen features and it would help with keeping vram requirements low for larger models.

### 🧰 Describe the solution you'd like
lightly_train.train_object_detection(
        out="out/my_experiment",
        model="dinov3/vitt16-ltdetr-coco",
        forzen_backbone=True,                        <-------- add something like
        data={
            "path": "my_data_dir",
            "train": "images/train",
            "val": "images/val",
            "names": {
                0: "person",
                1: "bicycle",
                2: "car",
                # ...
            },
        },
    )

**Comment by guarin** [comment]
Hi, thanks for the suggestion. Happy to add this! It should be relatively straight forward as we already have all the logic in place for some other models.

That being said, this will probably only work well when fine-tuning models that were already trained on an object detection task (e.g. COCO).

**Comment by Jaster1999** [comment]
My specific interest here is frozen finetuning the dinov3 backbones, due to their great dense features I was figuring that it may still work well (they talk about this in the dino paper when they finetune a plain detr on top of the frozen dino encoder they get very good results (albeit is the Dinov3 7B params model).

I'm also wondering whether the performance may be worse than a full finetune but the generalization to new slightly out of domain data would be better. Anyways awesome to hear it can be implemented!

**Comment by guarin** [comment]
This was released on 0.14.2
